src/App.js

```diff
@@ -8,12 +8,14 @@ import Profile from './pages/Profile';
 import Dragons from './pages/Dragons/Dragons';
 import Rockets from './pages/Rockets/Rockets';
 import { getMissions } from './redux/missions/missions';
+import { getDragons } from './redux/dragons/dragons';
 import { getRockets } from './redux/rockets/rockets';
 
 const App = () => {
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(getMissions());
+    dispatch(getDragons());
```

Comment:
Having dispatched the `getDragons()` action inside the `useEffect `here, there is no need to dispatch the same action in the `Dragons `component. This causes you to lose the reservation changes you made in the redux store when the page re-renders.